javascript-client/src/client.ts

```diff
@@ -78,6 +82,7 @@ export default class S3FFClient {
     for (const part of parts) {
       const chunk = file.slice(fileOffset, fileOffset + part.size);
       const response = await axios.put(part.upload_url, chunk, {
```

Comment:
```suggestion
      // eslint-disable-next-line no-await-in-loop
      const response = await axios.put(part.upload_url, chunk, {
```

If we don't set this globally, it needs to be ignored here.